sdcm/cluster.py

```diff
@@ -1406,6 +1406,8 @@ def config_setup(self, seed_address=None, cluster_name=None, enable_exp=True, en
                                          scylla_yaml_contents)
 
         if server_encrypt or client_encrypt:
+            localrunner.run('openssl x509 -req -in data_dir/ssl_conf/example/db.csr -CA data_dir/ssl_conf/cadb.pem -CAkey data_dir/ssl_conf/example/cadb.key -CAcreateserial  -out data_dir/ssl_conf/db.crt -days 365')
+            localrunner.run('openssl x509 -enddate -noout -in data_dir/ssl_conf/db.crt')
```

Comment:
Maybe worth to add try, except and if it fails to use the existing we have in the repo.

Reply:
fixed.